tests/test_invoice.py

```diff
@@ -193,6 +204,55 @@ async def make_assertion(*args, **_):
             protect_content=True,
         )
 
+        async def make_assertion(*args, **_):
+            kwargs = args[1]
```

Comment:
please move this to a new test `test_send_all_args_create_invoice_link` or so - i.e. independent of `send_invoice`